renovate.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+  "extends":[
+     "config:base"
+  ],
+  "timezone":"America/Los_Angeles",
+  "schedule":[
+     "after 3am on the 1st and 3rd day instance on Monday"
```

Comment:
just confirming my understanding..is this saying "every other Monday at 3:01am"?

Reply:
Yes, after 3am Pacific since we also added timezone (otherwise default would be UTC). 

I did see some examples where people also put end times in the schedule array, but I couldn't find anyone explaining why so figured this could be a good starting point!